src/components/display.js

```diff
@@ -0,0 +1,23 @@
+import PropTypes from 'prop-types';
+
+function Display({ total, next }) {
+  if (total) {
+    return (
+      <p className="screen flex">{total}</p>
+    );
+  } if (next) {
+    return (
+      <p className="screen flex">{next}</p>
+    );
+  }
+  return (
+    <p className="screen flex">0</p>
+  );
+}
```

Comment:
Well done on adding events using props and state. However, your display screen only shows one digit at a time and does not show whole arithmetic operations.

This is how you can code your `Display` component to resolve the issue:

![image](https://user-images.githubusercontent.com/38088818/198323887-fe968f5d-17f5-43db-8ecc-b0141c74ba60.png)

Now you can see all digits and operators being used in the calculation.

![image](https://user-images.githubusercontent.com/38088818/198324356-ece35b82-4b34-4050-bad8-55e0f4de2b77.png)